datafusion/sql/src/utils.rs

```diff
@@ -263,6 +263,37 @@ pub(crate) fn normalize_ident(id: Ident) -> String {
     }
 }
 
+pub(crate) fn normalize_value(value: &Value) -> Result<String> {
```

Comment:
`utils` is generally for things that are used across multiple files repeatedly.

How about we remove this function and just do the normalization in the `normalize` method under the `ValueNormalizer` struct?

If we separate the string conversion and normalization (as suggested in another comment), then we can move back the `value_to_string` fn too.

Reply:
Also, `exec_err` is only for Execution Errors. Code running in the planning stage should ideally use `plan_err`.

Reply:
This follows the origin code structure used by `normalize_ident`. However, I have removed this though.